packages/config/src/chainConfig/types.ts

```diff
@@ -40,6 +40,9 @@ export type ChainConfig = {
   // DENEB
   DENEB_FORK_VERSION: Uint8Array;
   DENEB_FORK_EPOCH: number;
+  // EIP6110 - Experimental fork
+  EIP6110_FORK_VERSION?: Uint8Array,
```

Comment:
no need to make it optional, in the corresponding mainnet spec, add it and set its epoch to `infinity`